index.html

```diff
@@ -64,5 +64,155 @@ <h1>
         </ul>
       </div>
     </section>
+
+    <!-- My recent Works Section -->
+
+    <section>
+      <div class="jobs" id="jobs-id">
+        <h1>My Recent Works</h1>
+        <article class="box-job">
+          <div class="box-img">
+            <img src="Imgs/Recent works/1.jpg" alt="" />
+          </div>
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul class="box-tags">
+            <li>React</li>
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="btn-showup">See Project</button>
+        </article>
+      </div>
+      <div class="jobs" id="jobs-id">
+        <article class="box-job">
+          <div class="box-img">
+            <img src="Imgs/Recent works/2.jpg" alt="" />
+          </div>
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul class="box-tags">
+            <li>React</li>
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="btn-showup">See Project</button>
+        </article>
+        <article class="box-job">
+          <div class="box-img">
+            <img src="Imgs/Recent works/3.jpg" alt="" />
+          </div>
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul class="box-tags">
+            <li>React</li>
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="btn-showup">See Project</button>
+        </article>
+        <article class="box-job">
+          <div class="box-img">
+            <img src="Imgs/Recent works/4.jpg" alt="" />
+          </div>
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul class="box-tags">
+            <li>React</li>
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="btn-showup">See Project</button>
+        </article>
+        <article class="box-job">
+          <div class="box-img">
+            <img src="Imgs/Recent works/5.jpg" alt="" />
+          </div>
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul class="box-tags">
+            <li>React</li>
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="btn-showup">See Project</button>
+        </article>
+        <article class="box-job">
+          <div class="box-img">
+            <img src="Imgs/Recent works/6.jpg" alt="" />
+          </div>
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul class="box-tags">
+            <li>React</li>
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="btn-showup">See Project</button>
```

Comment:
- [ ] Please implement the button interactions for all your buttons like :point_down: 
from figma 
![image](https://user-images.githubusercontent.com/97521557/234958364-14c5158f-4b02-484a-8595-6e741ba59cd7.png)